python/tvm/intrin.py

```diff
@@ -285,6 +285,22 @@ def trunc(x):
     return call_pure_intrin(x.dtype, "trunc", x)
 
 
+def abs(x):
+    """Get absolute value of the input element-wise.
+
+    Parameters
+    ----------
+    x : Expr
+        Input argument.
+
+    Returns
+    -------
+    y : Expr
+        The result.
+    """
+    return call_pure_intrin(x.dtype, "fabs", x)
```

Comment:
need to call abs c++ function here, instead of calling intrinsic

Reply:
Can you guide me, how can i make a call from tvm python to tvm cpp function? Can you give me any example?

Reply:
tvm.abs !!

Reply:
you will need to register functions, and they will appear on the registered namespace. Checkout e.g. https://github.com/dmlc/tvm/blob/master/src/api/api_ir.cc#L76 try register abs under make.abs, and call it from there